### DISCUSSION TITLE
onClick is not working

### DISCUSSION THREAD

**Discussion by SpBhalani** [vercel/next.js#54327] [Help] [answered]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 16.13.2
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.19
      eslint-config-next: 13.4.19
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.6
    Next.js Config:
      output: N/A
```

### Link to the code that reproduces this issue or a replay of the bug

https://github.com/SpBhalani/Netflix-clone-NextJs

### To Reproduce

NA

### Describe the Bug

onClick event is not getting triggered on any of the components.
Even I tried to console log the onChange event but it also didnt work.
In the given repo in auth path in component called card.tsx on the button component onClick is not working

### Expected Behavior

onClick should execute the function passed to the body of it.

**Comment by joerambo** [comment]
I cloned your repo and tested on macOS/chrome. onClick is firing correctly for me. Worked in dev and build. 

Can you create a blank next project and test onClick? That might help identify if it's an issue with your system (browser, extensions, etc).

**Comment by joerambo** [comment, marked as answer]
Check out: https://github.com/vercel/next.js/issues/54269#issuecomment-1685110362

They figured it out that node v16.14.0+ fixes this issue.

**Comment by SpBhalani** [comment]
Okay got it. Will upgrade to node 16.14.0 
Thanks!
